nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/components/administration/workflow/sidenav/manage-bucket/nf-registry-manage-bucket.js

```diff
@@ -462,6 +468,7 @@ NfRegistryManageBucket.prototype = {
                             if (!response.status || response.status === 200) {
                                 self.nfRegistryService.bucket = response;
                                 self.bucketname = self.nfRegistryService.bucket.name;
+                                self.description = self.nfRegistryService.bucket.description;
```

Comment:
There are similar code patters to this one at/about lines 530 and 589. Does "self.description" need to be added in these places as well?